internal/kafka/command_cluster_describe.go

```diff
@@ -160,6 +162,9 @@ func getKafkaClusterDescribeFields(cluster *cmkv2.CmkV2Cluster, basicFields []st
 		if cluster.Spec.Config.CmkV2Dedicated.EncryptionKey != nil && *cluster.Spec.Config.CmkV2Dedicated.EncryptionKey != "" {
 			describeFields = append(describeFields, "EncryptionKeyId")
 		}
+		if len(cluster.GetSpec().Config.CmkV2Dedicated.GetZones()) != 0 {
```

Comment:
nit: keep the same coding convention?

```suggestion
		if cluster.Spec.Config.CmkV2Dedicated.GetZones() != nil {
```

Reply:
Since we would avoid showing the zone field if it is empty, I think it might be better to check for len(GetZones()) != 0, which also covers the case of GetZones() != nil. I used `.GetSpec()` instead of `.Spec` to prevent the case of null pointer reference as `.GetSpec()` has a more robust null checking.

Reply:
You're right, will do